SIP-X.md

```diff
@@ -0,0 +1,75 @@
+---
+SIP: 'N/A'
+Title: Accepting ownership of Sovryn contracts
+Author: John Light (@john-light)
+Status: Draft
+Track: Contract
+Created: 2022-03-31
+---
+
+# SIP-X: Accepting ownership of Sovryn contracts  
+
+## Description  
+
+If approved, this proposal will result in an on-chain state change transferring the administrator role in the Sovryn AMM smart contracts from the Exchequer Multisig to the GovernorAdmin contract controlled by SOV stakers.
+
+Furthermore, this proposal will result in a signal from SOV stakers that they are willing and ready to accept the owner and/or adminstrator role in the following Sovryn smart contracts, to be transferred by the Exchequer Multisig no later than `2022-XX-XX 23:59:59 UTC`:
+
+|	Category        	| Contract name	              | Role  | New governor                                                 |
+| ----------------- |:---------------------------:|:-----:|:------------------------------------------------------------:|
+| LoanToken Modules |	                            | 	    |                                                              |
+|                   |	LoanTokenLogicLM            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanTokenLogicWRBTC         | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanTokenSettingsLowerAdmin | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+| Protocol          |	                            |       |                                                              |
+|                   |	Affiliates                  | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanClosingsLiquidation     | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanClosingsRollover        | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanClosingsWith            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanMaintenance             | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanOpenings                | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanSettings                | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	ProtocolSettings            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	SwapsExternal               | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
```

Comment:
These are irrelevant. The owner of the modules does not matter. These logic contracts are just providing logic. Whenever the protocol or a loan token contract is executed, the storage of the proxy contract is used and therefore also the owner of the proxy contract. The owner of the modules has no power at all.

Reply:
FYI - You can comment on multiple lines by clicking on the number of the first line you want to comment on and dragging it down to the last line you want to comment on :) With that having been said, which lines are you referring to here?

> Whenever the protocol or a loan token contract is executed, the storage of the proxy contract is used and therefore also the owner of the proxy contract. 

What does it mean for "the protocol or a loan token contract.." to be "executed"? What does it mean for "the storage of the proxy contract" to be "used and therefore also the owner of the proxy contract"? I'm having a hard time understanding this given my (lacking of?) knowledge of upgradeable contracts.

> The owner of the modules has no power at all.

Is there even an owner for these modules defined? If so, why, if they have no power? Would also repeat my [other question](https://github.com/john-light/sovryn/pull/7/files#r851281231) about the owner role here.

Reply:
- Storage on the logic contracts isn't used. Only the storage on the proxy is used.